src/components/mobileComponents/MobileHeader.tsx

```
@@ -18,13 +18,13 @@ interface MobileHeaderProps {
 }
 
 export default function MobileHeader({ isLogin }: MobileHeaderProps) {
-  const { data: userData, isLoading: userDataLoading } = useUserInfo();
+  const { data: userData } = useUserInfo();
   const { user } = useUserStore();
   // console.log(userData);
 
   const [isMenuOpen, setIsMenuOpen] = useState(false);
-  const router = useRouter();
   const toggleMenu = () => setIsMenuOpen((prev) => !prev);
+  const router = useRouter();
```

ESLint

[error] 'router' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)